.github/workflows/ci.yml

```diff
@@ -103,12 +103,8 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.7", "3.8", "3.9"]
-        nautobot-version: ["1.3.0", "1.4.2"]
-        # Also test Python 3.10 with Nautobot 1.3.x
-        include:
-          - python-version: "3.10"
-            nautobot-version: "1.4.2"
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        nautobot-version: ["1.4.2", "latest"]
```

Comment:
Very quickly 3.7 will be removed too...

Reply:
Yeah, I have it removed in my 2.0 branch already.